dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/process/ProcessImplForWin32.java

```diff
@@ -33,8 +34,17 @@
 import static com.sun.jna.platform.win32.WinBase.STILL_ACTIVE;
 
 public class ProcessImplForWin32 extends Process {
-    private static final sun.misc.JavaIOFileDescriptorAccess fdAccess
-            = sun.misc.SharedSecrets.getJavaIOFileDescriptorAccess();
+
+    private static final Field FD_ACCESS;
+
+    static{
+        try {
+            FD_ACCESS = FileDescriptor.class.getDeclaredField("fd");
+            FD_ACCESS.setAccessible(true);
+        } catch (NoSuchFieldException e) {
+            throw new RuntimeException(e);
```

Comment:
I'm unfamiliar with exception strategy in dolphin schedule so it would be better our committers can check whether the exception thrown obeys conventions.